HW1.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+# create a 2pi sequence and plug it in to sine function and cosine function
```

Comment:
Can we wrap the functionality in a main function?
`def main():
    # insert code here
    pass

if __name__ == "__main__":
    main()`